requirements_dev.txt

```diff
@@ -3,7 +3,7 @@ pip>=18.0
 wheel>=0.30.0
 
 # glotaran setup dependencies
-asteval==0.9.27
+asteval==0.9.28
```

Comment:
```suggestion
asteval==0.9.27
```

Reply:
This mimics the situation where a user has `0.9.28` already installed but then installs `pyglotaran` which installs a different version (right now `0.9.27`).

Reply:
Right, because requirements_dev.txt is first installed, and then the package dependencies ensures that the right version is installed (so not 0.9.28). Good thinking, will leave a note about it in the commit message.